docker-compose.yml

```diff
@@ -2,16 +2,16 @@ version: '2'
 
 services:
   frontend:
+    image: index.docker.io/buildkite/frontend
     build: .
-    volumes:
-      - "./bundle-analysis:/frontend/bundle-analysis"
-      - "./dist:/frontend/dist"
-      - "./coverage:/frontend/coverage"
     environment:
-      BUILDKITE:
-      BUILDKITE_COMMIT:
-      BUILDKITE_ORGANIZATION_SLUG:
-      BUILDKITE_PIPELINE_SLUG:
-      CI:
-      EMOJI_HOST:
-      FRONTEND_HOST:
+      - EMOJI_HOST
+      - FRONTEND_HOST
+      - BUILDKITE
+      - BUILDKITE_COMMIT
+      - BUILDKITE_ORGANIZATION_SLUG
+      - BUILDKITE_PIPELINE_SLUG
+      - CI
+    volumes:
+      - ".:/frontend:cached"
```

Comment:
Interesting!

I went away from recommending everyone just mount `.` into the container, and instead just mount the directories that have artifacts. Just to stop the volume mount clobbering the standard `ADD/COPY` files in the container's fs. Hence https://github.com/buildkite-plugins/docker-compose-buildkite-plugin#artifacts. Perhaps going back to `.` is a better idea?